app/Jobs/PushServerUpdateJob.php

```
@@ -141,65 +146,85 @@ public function handle()
 
         foreach ($this->containers as $container) {
             $containerStatus = data_get($container, 'state', 'exited');
-            $containerHealth = data_get($container, 'health_status', 'unhealthy');
-            $containerStatus = "$containerStatus ($containerHealth)";
+            $rawHealthStatus = data_get($container, 'health_status');
+            $containerHealth = $rawHealthStatus ?? 'unknown';
+            $containerStatus = "$containerStatus:$containerHealth";
             $labels = collect(data_get($container, 'labels'));
             $coolify_managed = $labels->has('coolify.managed');
```

PHPMD (2.15.0)

The variable $coolify_managed is not named in camelCase. (undefined)

(CamelCaseVariableName)